src/cljc/matcher_combinators/dispatch.cljc

```diff
@@ -15,6 +15,7 @@
 ;; equals base types
 (defn nil-dispatch [expected] (matchers/equals expected))
 (defn class-dispatch [expected] (matchers/equals expected))
+(defn object-dispatch [expected] (matchers/equals expected))
 (defn integer-dispatch [expected] (matchers/equals expected))
 (defn short-dispatch [expected] (matchers/equals expected))
 (defn long-dispatch [expected] (matchers/equals expected))
```

Comment:
Didn't those more specific dispatchers become obsolete due to the Object implementation?

Reply:
They are useful to leave in if you want to override the default matching behavior, such as to do rough numeric matching by default